packages/lending/src/protocols/compound-v3/configs.ts

```diff
@@ -13,6 +13,11 @@ export interface AssetConfig {
   supplyCap: string;
 }
 
+export interface PriceFeedConfig {
```

Comment:
MarketConfig could `extends PriceFeedConfig`?

Reply:
Missed that, thanks